core/templates/dev/head/services/StateTopAnswersStatsService.ts

```diff
@@ -163,10 +158,9 @@ oppia.factory('StateTopAnswersStatsService', [
        *    unresolved.
        */
       getUnresolvedStateStats: function(stateName) {
-        if (!stateTopAnswersStatsCache.hasOwnProperty(stateName)) {
-          throw Error(stateName + ' does not exist.');
-        }
-        return stateTopAnswersStatsCache[stateName].unresolvedAnswers;
+        return this.getStateStats(stateName).filter(function(answerStats) {
```

Comment:
I'm a bit confused. Looking at getStateStats() -- isn't this returning all answers, not just unresolved answers?

Reply:
Yup, but the filter removes any resolved answers

Reply:
Oh I see, sorry, misread. LGTM.